vulnerability/worker.go

```diff
@@ -96,11 +98,11 @@ func (f *VulnerabilityWorker) Run(ctx context.Context) {
 				f.log.Info("VulnerabilityWorker.work FetchInstances finished")
 			}()
 			f.wg.Add(1)
-			go func() {
+			go func(snapshots *[]ec2.EBSSnapshot) {
```

Comment:
Not a fan of this solution but just in order to move forward.

Reply:
Modified